stylesheets/components/card.css

```diff
@@ -0,0 +1,131 @@
+@import url(/stylesheets/abstract/variables.css);
+@import url(/stylesheets/abstract/color.css);
+@import url(/Stylesheets/main.css);
+
+.container {
+  margin-top: 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 2rem;
+}
+.card {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--light-grey);
+  margin-left: 2rem;
+}
+.card-doc {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--light-grey);
+  margin-left: 1rem;
+}
+.cardbox {
+  box-shadow: 7px 12px var(--dark-grey);
+}
+.card-only {
+  height: 11.5rem;
+  box-shadow: 20, 20 var(--bg-blue);
+}
+
+.card_image {
+  background-color: var(--bg-blue);
+  height: 10rem;
+  margin-bottom: 0.3rem;
+  background-size: cover;
+}
+
+.card_image2 {
+  height: 10rem;
+  width: 10rem;
+  background-size: cover;
+}
+
+.card-1 {
+  background-image: url("https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8c2hvZXxlbnwwfHwwfHw%3D&w=1000&q=80");
+}
```

Comment:
You can use this in HTML itself.

Reply:
ya but i think its ll be ok if i did here also.

Reply:
Definitely, it's okay, although it's good practice to put images in Html itself and change its styles on CSS if required.

Reply:
okay i willl do it . ty for good observation